SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community vulnerability and Security reporting
+
+## Report a Vulnerability or Security issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers.
+
+## How to report a security vulnerability
+To file a vulnerability report please send and e-mail to the private eiffel-community-security@googlegroups.com  list. The e-mail should list the security specific details as well as the [standard bug report information](https://github.com/eiffel-community/.github/blob/master/.github/ISSUE_TEMPLATE.md). Only the community security officers will have access to e-mails sent on the security and vulnerability list.
+
+Triage and handling of the vulnerability report will be conducted within one week. If the vulnerability severity and impact is high a patch will be published with urgency.
+
```

Comment:
If the project maintainers/contributors identify such issues themselves, how they should handle it? (the process taking the shortcut we talked about during the TC meeting)

They could perhaps still notify the security officer/community security volunteers once the issue is fixed so it can be listed on the security page together with other issues.

Reply:
> If the project maintainers/contributors identify such issues themselves, how they should handle it? (the process taking the shortcut we talked about during the TC meeting)
> 
> They could perhaps still notify the security officer/community security volunteers once the issue is fixed so it can be listed on the security page together with other issues.

I did not fully understand why the maintainers would have the privilege to shortcut the process, and also since the process is light weight it shouldn't really be a problem to follow it. I'm actually leaning towards "same process for all".

Reply:
This was based on the discussion we had during the meeting on Monday. But I agree with you, they should follow the process.

But, I still think this, _projects themselves following the process_, should somehow be stated so we don't miss the issues found & fixed by the projects themselves. Otherwise, those issues will be buried deep in the history, mixed with the regular development activities, making them invisible to the users who may be using vulnerable version.

Reply:
Agreed, I'll add something to clarify.

Reply:
Added some clarification on this.